common/test/acceptance/tests/studio/test_studio_settings_details.py

```diff
@@ -68,7 +68,7 @@ def test_prerequisite_course_save_successfully(self):
 
         # Refresh the page to load the new course fixture and populate the prrequisite course dropdown
```

Comment:
I'll fix the comments shortly -- per @jzoldak there's a difference in how page.visit() works versus the custom logic that was being executed